BrainPortal/public/swagger/cbrain-6.2.0.1-swagger.json

```diff
@@ -517,6 +517,30 @@
             "description" : "No session created yet."
           }
         }
+      },
```

Comment:
Please undo all changes to this JSON file. Instead, provide a NEW pair of YAML and JSON file with a new version number (e.g. 6.2.0.2). We also need to publish this on SwaggerHub